### PR TITLE
Create a shtldr's file

### PR DESCRIPTION
**Pull request by shtldr**
Concludes issue #9 by adding file shtldr-hometown.md

@krasmar Nazdar :+1:

### CODE REVIEW

**Comment by Jesusc83** [comment]
@shtldr  :+1:  Good work.. 
Can you add more lines in your file. Thanks...

**Comment by shtldr** [comment]
@Jesusc83 Done

**Comment by krasmar** [comment]
@shtldr a vy jste kdo, pane?

File: shtldr-hometown.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+- Restaurant El Emir
 
 ## Places to visit
 - Charles' Bridge
 - Prague Castle
+- Petrin look-out
 
 ## Places to avoid
 - Palmovka
+- Park near Muzeum
```
Comment (krasmar) [inline shtldr-hometown.md]: which one? :scream:

Comment (shtldr) [inline shtldr-hometown.md]: The one on the left, leading to Vinohrady

**Pull request merged**